modules/ps_mainmenu/ps_mainmenu.tpl

```diff
@@ -1,40 +1,30 @@
-{assign var=_counter value=0}
 {function name="menu" nodes=[] depth=0 parent=null}
   {if $nodes|count}
-    <ul class="{if $depth == 0}main-menu__tree{else} ms-2{/if}" {if $depth == 0}id="top-menu" {/if} data-depth="{$depth}">
+    <ul class="{if $depth == 0}main-menu__tree{/if}{if $depth == 1} row g-3{/if}{if $depth > 1} submenu{/if}"
+      {if $depth == 0}id="top-menu" {/if} data-depth="{$depth}">
       {foreach from=$nodes item=node}
-        <li class="{$node.type}{if $node.current} current {/if}{if $depth == 0} main-menu__tree__item{/if}"
+        <li
+          class="{$node.type}{if $node.current} current {/if}{if $depth == 0} main-menu__tree__item{/if}{if $depth == 1} col-4{/if}"
```

Comment:
```suggestion
          class="{$node.type}{if $node.current} current {/if}{if $depth == 0} main-menu__tree__item{/if}{if $depth == 1} col-3{/if}"
```

Reply:
I prefer 4 cols rather than 3, but figma prototype shows 3, so that's why y use col-4